### PR TITLE
Add Jriver Media Center 22.0.54

### PR DESCRIPTION
**Pull request by nrlquaker**
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

- [x] Named the cask according to the [token reference]. (not sure if I used the correct name according to your token reference. Jriver Media Center is full name mentioned at homepage. But appname is Media Center. But its very generic.)
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

### CODE REVIEW

File: Casks/jriver-media-center.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask 'jriver-media-center' do
+  version '22.0.54'
+  sha256 'fc85bf66f75ce640936f1e21fc11c0786f2b43811f2082a4a4984b42676c337b'
+
+  url 'http://files.jriver.com/mediacenter/channels/v22/stable/MediaCenter220054.dmg'
```
Comment (miccal) [inline Casks/jriver-media-center.rb]: Please change this to `url "http://files.jriver.com/mediacenter/channels/v#{version.major}/stable/MediaCenter#{version.no_dots}.dmg"`.

File: Casks/jriver-media-center.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  name 'JRiver Media Center'
+  name 'JRiver'
+  name 'Media Center'
+  homepage 'https://www.jriver.com/'
+
+  app 'Media Center 22.app'
```
Comment (miccal) [inline Casks/jriver-media-center.rb]: Please change this to `app "Media Center #{version.major}.app"`.

File: Casks/jriver-media-center.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+cask 'jriver-media-center' do
+  version '22.0.54'
+  sha256 'fc85bf66f75ce640936f1e21fc11c0786f2b43811f2082a4a4984b42676c337b'
+
+  url 'http://files.jriver.com/mediacenter/channels/v22/stable/MediaCenter220054.dmg'
+  name 'JRiver Media Center'
+  name 'JRiver'
+  name 'Media Center'
```
Comment (miccal) [inline Casks/jriver-media-center.rb]: Please remove the last two `name`'s; as they are included in the first `name`.

**Review by miccal — CHANGES_REQUESTED** [review]

**Comment by nrlquaker** [comment]
Does someone know why build is failing?

> Error: undefined method `major' for nil:NilClass

**Comment by miccal** [comment]
Just a spelling mistake - I have fixed it.

**Comment by vitorgalvao** [comment]
Verified locally due to Travis delays.

**Pull request merged**